ckanext/datagovtheme/templates/base.html

```diff
@@ -4,6 +4,7 @@
     {{ super() }}
     {% block googleanalytics_custom_gtag %}
       {% include '/snippets/gtm/gtm_head.html' %}
+      {% include '/snippets/gtm/gtm_datalayer.html' %}
```

Comment:
Does the datalayer need to go before the gtm setup?

Reply:
Could be, I dunno. Moved.